Updates for PM review.

mcherem opened the pull request:
PR for the second half of the PM's doc review.

mcherem committed:
Updates for PM review.

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -5,7 +5,7 @@ endif::[]
 
 
 Anypoint Partner Manager allows you to manage a large number of trading partners and their message flows. A message flow is the path along which a message travels, from its source, through a transformation (via a map), and to its target. You'll need to perform some tasks before you can begin transmitting messages with your trading partners. A transmission is a single communication event where a t ...
```
allows > enables
This is per our style reference. Do not let your reviewers revert.

kbostrom replied:
Suggest breaking para at "You'll need to...."

kbostrom replied:
where > during which

kbostrom replied:
"A transmission is a single communication event where a transfer of a payload which may contain one or more messages."  >
"A transmission is a single communication event during which payload transfer sometimes requires more than one message."

The original sentence is not grammatical. Do not let reviewers revert.

mcherem replied:
Fixed.

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -5,7 +5,7 @@ endif::[]
 
 Anypoint Partner Manager allows you to manage a large number of trading partners and their message flows. A message flow is the path along which a message travels, from its source, through a transformation (via a map), and to its target. You'll need to perform some tasks before you can begin transmitting messages with your trading partners. A transmission is a single communication event where a t ...
-You set yourself up as a host partner, then configure your partners, and then create message flows between you and your partners. 
+You set yourself up as a host partner, then configure your partners, and then create message flows between you and your partners.
```
This sentence should logically follow the sentence about needing to perform some tasks...those tasks being "First, you set yourself as host partner. Then you configure your partners, and finally create message flows between you and your partners."

Suggest following with "Even though a transmission is a single communication event, it sometimes requires more than one message (or transaction set) to transfer payload."

The original sentences sound choppy, without a logical flow or transitions between them. Unless this affects technical meaning, I suggest that you not revert.

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -15,19 +15,15 @@ Use this checklist to help organize the information you need to gather before yo
 
```
This introduction is not consistent with the text that follows it. It mentions info from partners and then understanding your backend, but then it describes these things in the reverse order. The two should match..

kbostrom replied:
The final sentence above is unnecessary. What follows is not a checklist; it describes what the previous two sentences introduced. Suggest omitting, especially since the sentence begins with "Use this..." and does not end with a colon pointing to anything.

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -15,19 +15,15 @@ Use this checklist to help organize the information you need to gather before yo
 
 Information you'll need to have on hand as the host organization:
```
"on hand" is colloquial and difficult to translate.
If this is the aforementioned "understanding of how your backend systems work," I suggest rewording either that or this so that they match.
It seems more logical to change the introduction to state simply that you need certain information about your system as host, and your partners' systems: 

Information you need about your organization:

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -15,19 +15,15 @@ Use this checklist to help organize the information you need to gather before yo
 Information you'll need to have on hand as the host organization:
 
-
 * Your organization's identifiers (such as the AS2 Identifier, X12 ISA, X12 GS, and so on)
```
AS2 Identifier > AS2 ID

I looked this up and saw other versions as well: AS2 System Identifier, for one. What was standard was use of AS2 ID, which is parallel with the other items in this list.

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -15,19 +15,15 @@ Use this checklist to help organize the information you need to gather before yo
 * Your organization's identifiers (such as the AS2 Identifier, X12 ISA, X12 GS, and so on)
 
-
 * Endpoints: There are three types of endpoints within AnyPoint Partner Manager -
```
QUESTION: This list is about host info, but it looks like one of these endpoints is actually partner info (acknowledgement endpoint)?
If so, suggest changing this bullet item to...

   * Your receive and target endpoints:
      **Receive endpoints
         Exposed to your trading partners and used by your organization to receive transmissions from them
     **Target endpoints
        Host-internal endpoints to which you route messages from your trading partners, after those messages
        are transformed in the message flow

mcherem replied:
I'm gonna need to check with the PM on the usage here.

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -15,19 +15,15 @@ Use this checklist to help organize the information you need to gather before yo
-
 * Endpoints: There are three types of endpoints within AnyPoint Partner Manager -
 
+** Receive endpoints - On the Host side, for an inbound transmission you need endpoints your organization uses to receive transmissions from your partners. These endpoints are resources your organization exposes to your trading partners.
```
See previous suggestion.
If you keep this text, these copyedits:

```suggestion
** Receive endpoints
    As host, endpoints your organization uses to receive transmissions from your partners. These endpoints are resources your organization exposes to your trading partners.
```

mcherem replied:
Again, from previous.

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -15,19 +15,15 @@ Use this checklist to help organize the information you need to gather before yo
-- Target endpoints - As the Host, your organization has internal endpoints where you route messages from the Partner, after they are transformed within the message flow.
-
-- Acknowledgement endpoints - Your partners provide you with endpoint configurations that can be used to send acknowledgement messages like 997s or 999s back to them. Acknowledgements are typically sent back to trading partners corresponding to one of more B2B messages you may receive from them.
+** Target endpoints - As the Host, your organization has internal endpoints where you route messages from the Partner, after they are transformed within the message flow.
```
See previous suggestion.
If you keep this text, these copyedits:

```suggestion
** Target endpoints
    As host, internal endpoints to which you route messages from the partner, after those messages are transformed within the message flow.
```

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -15,19 +15,15 @@ Use this checklist to help organize the information you need to gather before yo
-- Acknowledgement endpoints - Your partners provide you with endpoint configurations that can be used to send acknowledgement messages like 997s or 999s back to them. Acknowledgements are typically sent back to trading partners corresponding to one of more B2B messages you may receive from them.
+** Target endpoints - As the Host, your organization has internal endpoints where you route messages from the Partner, after they are transformed within the message flow.
 
+** Acknowledgement endpoints - Your partners provide you with endpoint configurations that can be used to send acknowledgement messages like 997s or 999s back to them. Acknowledgements are typically sent back to trading partners corresponding to one of more B2B messages you may receive from them.
```
This belongs in the second list, of information that the host needs to obtain from the partner.
Suggest moving, with these copyedits:

```suggestion
* Partner acknowledgment endpoints
  Your partners provide you with endpoint configurations that you use as targets for sending messages (like 997s or 999s) to trading partners, acknowledging receipt of their transmissions.
```

mcherem replied:
I'm going to have to review this with the PM. If it sounds confusing, it's because as the host, you do some partner configurations you need to communicate before you configure a partner. They kind of reuse the UI in a way that makes it a bit complicated.

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -37,7 +33,6 @@ Information you'll need to have on hand as the host organization:
 * Any Dataweave transformation maps your organization uses for B2B
 
-
 Information needed for each trading partner:
```
```suggestion
Information you need about each trading partner:
```

kbostrom commented on modules/ROOT/pages/B2B-overview.adoc:
```diff
@@ -37,7 +33,6 @@ Information you'll need to have on hand as the host organization:
-
 Information needed for each trading partner:
 
 * Partner name (usually the name of the company)
```
For text below, which I cannot alter:

   * Partner identifiers (such as AS2 ID, ....)
   * Partner acknowledgment endpoints
     [[see above]]
   * Electronic...
   * Message types to use
      For example....
   * Partner digital certificates....

kbostrom commented on modules/ROOT/pages/_partials/endpoint-http-receive.adoc:
```diff
@@ -2,14 +2,13 @@
-
-=== Global Config
+=== Global Configuration
 
```
Suggest adding a text transition between the subtitle and the table.
Suggest converting table to a definition list. See style ref.

mcherem replied:
Will work on this for next draft.

kbostrom commented on modules/ROOT/pages/_partials/endpoint-http-receive.adoc:
```diff
@@ -2,14 +2,13 @@
-
-=== Global Config
+=== Global Configuration
 
```
Suggest adding a text transition between subtitle and table.
Suggest converting table to definition list; see style ref.

mcherem replied:
Will work on for next draft.

kbostrom commented on modules/ROOT/pages/_partials/endpoint-http-receive.adoc:
```diff
@@ -2,14 +2,13 @@
 |Name |Description
 |Protocol |*HTTP* or *HTTPS* +
-Type: value representing a Radio-button selection 
+Type: value representing a Radio-button selection
```
```suggestion
Type: Value representing a radio-button selection +
```

mcherem replied:
Next draft.

kbostrom commented on modules/ROOT/pages/_partials/endpoint-http-receive.adoc:
```diff
@@ -2,14 +2,13 @@
 |Protocol |*HTTP* or *HTTPS* +
-Type: value representing a Radio-button selection 
+Type: value representing a Radio-button selection
 Required: yes +
```
```suggestion
Required: Yes +
```

mcherem replied:
Next draft.

kbostrom commented on modules/ROOT/pages/_partials/endpoint-http-send.adoc:
```diff
@@ -10,7 +10,7 @@ These settings are a subset of those in xref:4.0@mule-runtime::http-request-conn
 |Protocol
 |Select *HTTP* or *HTTPS* +
-Type: Radio-button-selected value +
+Type: selected value +
```
```suggestion
Type: Selected value +
```

kbostrom commented on modules/ROOT/pages/_partials/endpoint-http-send.adoc:
```diff
@@ -52,7 +52,7 @@ Required: no +
 
 |Method |&nbsp; +
-Type: Listbox selection +
+Type: selection +
```
```suggestion
Type: Selection +
```

kbostrom commented on modules/ROOT/pages/_partials/endpoint-http-send.adoc:
```diff
@@ -52,7 +52,7 @@ Required: no +
 |Method |&nbsp; +
-Type: Listbox selection +
+Type: selection +
 Required: no +
```
```suggestion
Required: No +
```

mcherem replied:
Fixed.

kbostrom commented on modules/ROOT/pages/activity-tracking.adoc:
```diff
@@ -13,7 +13,7 @@ Uses for this data include:
 * Auditing
 
-Anypoint Partner Manager allows you to monitor B2B transmission activity.
+Anypoint Partner Manager allows you to monitor trading partner transmission activity.
```
```suggestion
Anypoint Partner Manager enables you to monitor trading partner transmission activity.
```

kbostrom replied:
GLOBALLY standardize how this document refers to...:

     host, host organization
     partner, trading partner

mcherem replied:
Fix in next draft.

kbostrom commented on modules/ROOT/pages/activity-tracking.adoc:
```diff
@@ -13,7 +13,7 @@ Uses for this data include:
 
-Anypoint Partner Manager allows you to monitor B2B transmission activity.
+Anypoint Partner Manager allows you to monitor trading partner transmission activity.
 Activity is the full set of end-to-end events involved in processing a transmission such as transmission metadata, messages, and errors.
```
```suggestion
Activity is the full set of end-to-end events involved in processing a transmission, such as transmission metadata, messages, and errors.
```
...but how are metadata, messages, and events considered "events"? Needs another word.

mcherem replied:
Rewritten to:
Activity is the full set of end-to-end events related to transmission metadata, messages, and errors as they are involved in processing a transmission.

kbostrom commented on modules/ROOT/pages/activity-tracking.adoc:
```diff
@@ -13,7 +13,7 @@ Uses for this data include:
+Anypoint Partner Manager allows you to monitor trading partner transmission activity.
 Activity is the full set of end-to-end events involved in processing a transmission such as transmission metadata, messages, and errors.
 
 You can:
```
This is an insufficient amount of text to transition from one paragraph to the next.
Do we mean "Using Partner Manager, you can perform the following tasks:" or "...achieve the following goals:"?

kbostrom replied:
Is line 26 below supposed to be another list item in the list above it?
There is a content gap between this list and the TASK "Track Activity for Message Flows" (no colon).

kbostrom replied:
In line 30 below, boldface **Activity**, because it is the label of a UI item that the user selects.

Also change "left-hand" to just "left."

kbostrom commented on modules/ROOT/pages/configure-host.adoc:
```diff
@@ -6,11 +6,13 @@ endif::[]
 Your organization is considered the host partner in Anypoint Partner Manager.
```
ditto standardizing how this doc refers to host and partner throughout

kbostrom commented on modules/ROOT/pages/configure-host.adoc:
```diff
@@ -6,11 +6,13 @@ endif::[]
 Your organization is considered the host partner in Anypoint Partner Manager.
 Within the Host page you can do the following:
```
```suggestion
Within the Host page, you can do the following:
```

kbostrom commented on modules/ROOT/pages/configure-host.adoc:
```diff
@@ -6,11 +6,13 @@ endif::[]
+* Add storage for your B2B message payloads
 
 
 == Configure the host
```
```suggestion
== Configure the Host
```

kbostrom replied:
Suggest adding a text transition between the subtitle and the first step.

kbostrom commented on modules/ROOT/pages/configure-host.adoc:
```diff
@@ -19,16 +21,17 @@ Within the Host page you can do the following:
 
 . Select the name of your organization from the navigation pane on the left-hand side.
```
left-hand > left

kbostrom commented on modules/ROOT/pages/configure-host.adoc:
```diff
@@ -19,16 +21,17 @@ Within the Host page you can do the following:
 
-. From here you configure the following based on your indentifier and certificate type as well as any global settings you need to change and the location of your payload storage configuration.
-Refer to X12 Settings to change any of the default settings for all X12 received messages.
+From here you configure the following based on your indentifier and certificate type. This includes any global settings you need to modify, and the location of your payload storage configuration.
```
```suggestion
. Configure the following based on your identifier and certificate type, including any global settings you need to modify and the location of your payload storage configuration:
```

kbostrom commented on modules/ROOT/pages/configure-host.adoc:
```diff
@@ -19,16 +21,17 @@ Within the Host page you can do the following:
+
+// Refer to X12 Settings to change any of the default settings for all X12 received messages.
 
 * Add your identifiers such as the AS2 Identifier, X12 ISA, X12 GS, and/or DUNS.
```
AS2 ID

and/or > [choose just one]

kbostrom commented on modules/ROOT/pages/configure-host.adoc:
```diff
@@ -19,16 +21,17 @@ Within the Host page you can do the following:
 * Change any of the default global X12 validation, acknowledgement, control number, and character set settings.
 
-* Storage for your B2B message payloads. Configure the location where you store payloads containing the messages you send to and receive from partners.
+* Storage for your B2B message payloads. Configure the location where you store payloads containing the messages you send to and receive from your partners.
```
This bullet list item is not parallel with its peers, which are all verb-based. Reword for parallelism.

kbostrom commented on modules/ROOT/pages/configure-host.adoc:
```diff
@@ -19,16 +21,17 @@ Within the Host page you can do the following:
-* Storage for your B2B message payloads. Configure the location where you store payloads containing the messages you send to and receive from partners.
+* Storage for your B2B message payloads. Configure the location where you store payloads containing the messages you send to and receive from your partners.
 
 When you are ready, you can Configure your trading partner(s).
```
Not "when you are ready"; rather, after you configure yourself as host, you can configure your trading partners.

kbostrom commented on modules/ROOT/pages/configure-message-flows.adoc:
```diff
@@ -5,20 +5,24 @@ endif::[]
 
 To receive messages from a partner, create a message flow and configure the sections for both the source and target.
```
I have already edited this text. I see that my edits were not implemented, so I am not going to repeat that work.
EDITS SHOULD BE IMPLEMENTED UNLESS THEY CHANGE TECHNICAL MEANING.
I will only copyedit the changed text from here on in this file. It is a waste of my time to repeat edits reverted by other reviewers.

kbostrom commented on modules/ROOT/pages/configure-message-flows.adoc:
```diff
@@ -5,20 +5,24 @@ endif::[]
 To receive messages from a partner, create a message flow and configure the sections for both the source and target.
 For the Endpoint, source at host - this is the origin of the message flow when defined by a host endpoint and message type, often an API.
-For the endpoint, target at host - this is the final destination of a message flow when defined by a host endpoint and message type, often a Host API.
+For the Endpoint, target at host - this is the final destination of a message flow when defined by a host endpoint and message type, often a Host API.
```
What is this, a list? Where are the bullets? Why is endpoint capitalized?

Bullet list items should take this format:

* This thing
   Description of thing

kbostrom replied:
Host API > host API

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -11,9 +11,7 @@ From the Partners page you can:
-
-* Add identifiers to uniquely identify them from one another.
-
+* Add identifiers to uniquely identify them from one another
```
These list items are complete sentences and should end with periods.

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
-. Click the *New Partner* button and proceed to:
-- Add your trating partners name, and
-- Select an Identifier from the *Type* drop-down menu. This uniquely identifies you as a trading partner. For example, depending on your setup, select one of the following: AS2, X12 ISA, X12 GS, or the DUNS number.
+. Click the *New Partner* button.
```
```suggestion
. Click *New Partner*.
```
Per style reference.

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+. Add your trading partner's name.
 
-. Click *Create Partner* to create your trading partner. at this point, your trading partner has been created within the Anypoint platform and you may proceed to setup message flows related to this partner. 
+. Select an Identifier from the *Type* drop-down menu. This uniquely identifies you as a trading partner. For example, depending on your setup, select one of the following: AS2, X12 ISA, X12 GS, or the DUNS number.
```
```suggestion
. Select an identifier from the *Type* menu. 
  This uniquely identifies you as a trading partner. For example, depending on your setup, select one of the following: AS2, X12 ISA, X12 GS, or the DUNS number.
```
How many times does this doc repeat these identifiers? Evaluate and try to reduce.

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
-. Click *Create Partner* to create your trading partner. at this point, your trading partner has been created within the Anypoint platform and you may proceed to setup message flows related to this partner. 
+. Select an Identifier from the *Type* drop-down menu. This uniquely identifies you as a trading partner. For example, depending on your setup, select one of the following: AS2, X12 ISA, X12 GS, or the DUNS number.
+
+. Click *Create Partner* to create your trading partner. At this point, your trading partner has been created within the Anypoint platform, so you can go to setup message flows related to this partner.
```
```suggestion
. Click *Create Partner*
  Your trading partner is created within the Anypoint Platform and you can set up message flows to this partner.
```

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
 
 == Update Trading Partner Profile
 
+You can also add more details related to this trading partner. You can add the following information for each trading partner from the *Partners* selection in the left-hand navigation:
```
Replace this with a short description that says that you can add more detail to your trading partner's profile, and tell WHY your audience might want or need to do this.

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
 
+You can also add more details related to this trading partner. You can add the following information for each trading partner from the *Partners* selection in the left-hand navigation:
+
+. Click the *About* button.
```
```suggestion
. From the left navigation panel, select *Partners*.
. Click *About*.
```

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+
+. Click the *About* button.
+
+. Add the following by clicking the *Edit* button under the *About* section:
```
```suggestion
. Click *Edit* and provide values for the items shown.
```
I suggest omitting all of the following, which merely documents the UI, which we typically do not want to do. Will the audience not understand how to provide values for these?

If you decide to keep this list, then minimize it:

. Click **Edit** and provide the following information:
   * Description
   * Website
   * Logo
     Upload only JPEG or PNG files that are smaller than 25KB.
   * Corporate address
   * Add[itional?] contacts

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
-- Logo - upload a logo associated to your trading partner. jpeg and png files are acceptable formats for the logo. Please ensure the logo file is smaller than 25kb.
-- Corporate address - provide an address comprsing of the street, City, State, Postal code, and Country.
-- Add contacts - Options are available to add 'business contacts', 'technical contacts' or 'other contacts'. You may add up to 3 contacts associated to a trading partners profile.
+. Click on the *Save* button after providing all the necessary information. When the trading partner's profile is successfully updated, you receive a confirmation message.
```
```suggestion
. Click *Save*.
  You receive a confirmation message.
```

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+. Click on the *Save* button after providing all the necessary information. When the trading partner's profile is successfully updated, you receive a confirmation message.
 
-Click on the *Save* button when you have provided all the necessary information. When the trading partner's profile is successfully updated you will receive a confirmation message indicating 'Partner "<your trading partners name>" updated'
+. Add more identifiers for your Trading partner by clicking the *New* button within the *Identifiers* section. Select an appropriate identifier type and add an identifier value as provided by your trading partner.
```
Why is there a step after the user saves the profile?
This sounds like it should be a different task.
Note also that this is at least two and maybe three steps rather than one.

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+. Add more identifiers for your Trading partner by clicking the *New* button within the *Identifiers* section. Select an appropriate identifier type and add an identifier value as provided by your trading partner.
 
-* Add more identifiers for your Trading partner. To add more identifiers, click on the *New* button within the 'Identifiers' section. Select an appropriate identifier type and provide a the identifier value as provided by your trading partner. after you have provided all the information needed, click on the *Create identifier* button. When an identifier is successfully created you will receive a  ...
+. Click the *Create identifier* button to save your changes. When an identifier is successfully created you receive a confirmation message.
```
```suggestion
. Click *Create identifier* to save your changes. 
  You receive a confirmation message.
```

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+. Click the *Create identifier* button to save your changes. When an identifier is successfully created you receive a confirmation message.
 
-* Your partner's digital certificate (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol). To add a certificate, click on the *New* button within the 'Certificates' section. Upload the digital certificate (typically a .pem file) provided by your trading partner, and proceed to click on *Upload certificate* button. When a certificate is successfully uploaded you will ...
+. To add a certificate, click the *New* button within the 'Certificates' section. This is your partner's digital certificate (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
```
Another new task?
This set of tasks needs to be introduced at its beginning with text that identifies each of the different phases of this process of configuring a partner. Which tasks/steps are required? Which are additional? When does the user want or need to perform the optional?
Separate different goals into different tasks, and make sure the steps are brief and clear, documenting just one user action in the UI.
If a step is followed by more than one line or two of conceptual (reasons) or reference (lists of things), then you need to reorganize.
A good indicator is also if you have to go to more than two levels of step or list item.

kbostrom replied:
Copyedit only:
```suggestion
. To add a certificate, click *New* in the Certificates section. 
  This is your partner's digital certificate (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
```

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
-* Your partner's digital certificate (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol). To add a certificate, click on the *New* button within the 'Certificates' section. Upload the digital certificate (typically a .pem file) provided by your trading partner, and proceed to click on *Upload certificate* button. When a certificate is successfully uploaded you will ...
+. To add a certificate, click the *New* button within the 'Certificates' section. This is your partner's digital certificate (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
+
+. Upload the digital certificate (typically a .PEM file) provided by your trading partner by clicking the *Upload certificate* button. When a certificate is successfully uploaded you receive a confirmation message.
```
Copyedits only:
```suggestion
. Upload the digital certificate (typically a .PEM file) provided by your trading partner by clicking *Upload certificate*. 
You receive a confirmation message.
```

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+
+. Upload the digital certificate (typically a .PEM file) provided by your trading partner by clicking the *Upload certificate* button. When a certificate is successfully uploaded you receive a confirmation message.
+
+. Configure any global validation settings if you need to override a default value for a particular partner.
```
Copyedits only:
```suggestion
. Configure global X12 validation settings to override a default value for a particular partner.
```

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+. Upload the digital certificate (typically a .PEM file) provided by your trading partner by clicking the *Upload certificate* button. When a certificate is successfully uploaded you receive a confirmation message.
+
+. Configure any global validation settings if you need to override a default value for a particular partner.
+
```
The remainder of this text about global settings does not belong in the middle of a task topic.
Make a separate concept topic about these settings and cross-refer to it in See Also for this file.

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+
+Global X12 setting include:
+
+* validations
```
```suggestion
* Validations
```

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+* Control numbers
+* Character sets
+
+Refer to the following options for each here.
```
omit

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -25,32 +23,55 @@ From the Partners page you can:
+Refer to the following options for each here.
+
+=== Validations
+Apply these to your partners.
```
When? Why?

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -119,7 +140,9 @@ Default::
 
-- Control Numbers: Please apply validations related to your partners / your use of control numbers within the X12 message.
+=== Control Numbers
+Apply  validations related to your partners / your use of control numbers within the X12 message.
```
```suggestion
Apply validations related to your partners or your use of control numbers within the X12 message.
```

kbostrom replied:
When? Why?

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -152,7 +175,10 @@ Default::
-- Character Sets: Selected charater set and character encoding options pertaining to your partners / your X12 message.
+=== Character Sets
+
+Selected character set and encoding options pertaining to your partners and/or your X12 message.
```
```suggestion
Select character set and encoding options pertaining to your partners and your X12 messages.
```

kbostrom replied:
When? Why?

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -189,7 +215,7 @@ Default::
 * To find a partner, search the Partners list by name, and select the name of the partner
 
-Refer to the X12 Settings to change any of the default settings for all X12 received messages.
+// Refer to the X12 Settings to change any of the default settings for all X12 received messages.
```
So does this separate settings topic already exist?

kbostrom commented on modules/ROOT/pages/configure-partner.adoc:
```diff
@@ -189,7 +215,7 @@ Default::
-Refer to the X12 Settings to change any of the default settings for all X12 received messages.
+// Refer to the X12 Settings to change any of the default settings for all X12 received messages.
 
 When you are ready, you can Configure Message Flows.
```
GLOBALLY omit these transitions at the end of files. The topic set overview and topic presentation order should make clear the chronology.

kbostrom commented on modules/ROOT/pages/document-types.adoc:
```diff
@@ -7,13 +7,38 @@ Configuring a message type for a partner means configuring settings specific to
 
 == X12
```
This file does not look ready for review. The presentation needs work. NOT EDITED.

amitfernandes committed:
minor updates

kbostrom commented on modules/ROOT/pages/endpoints.adoc:
```diff
@@ -39,6 +39,7 @@ Valid values::
 
```
This file both contradicts earlier text about the three endpoint types and appears to be incomplete. 
The presentation needs work.
NOT EDITED.
Also note: if you intend to have Reference topics like these, then you need not include this information in the main text body as well, but rather refer to these references by using See Also text in the concept or task topics to which they apply.

kbostrom commented on modules/ROOT/pages/index.adoc:
```diff
@@ -12,8 +12,8 @@ Enterprises use Anypoint Partner Manager to carry out business transactions such
 They are typically transmitted over transports (such as HTTP(s), SFTP, and exchanges such as AS2) and use message formats such as X12 and JSON.
```
Is this an older version of the beginning of this set?
Please send me only the finished whole. Do not include deprecated files.
NOT EDITED

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -6,22 +6,27 @@ endif::[]
 Set up Partner Manager to be used by different people in your organization: message flow developers, operations managers, and IT managers.
```
Where does this file fit in the doc chronology? Is this introduced in the introduction? I don't recall seeing this task in what is covered by this doc.

kbostrom replied:
This belongs at the beginning of this doc. Why is it here? Did you just add it?

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -6,22 +6,27 @@ endif::[]
 Set up Partner Manager to be used by different people in your organization: message flow developers, operations managers, and IT managers.
 To configure the system, you should be familiar with these concepts before setting up Partner Manager:
```
```suggestion
You should be familiar with these concepts before setting up Partner Manager:
```

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -6,22 +6,27 @@ endif::[]
+
+* Dataweave transformations
+
+* Digital Certificates such as X.509 certificates in PKCS12 file format for the AS2 transport protocol
```
```suggestion
* Digital certificates, such as X.509 certificates in PKCS12 file format for the AS2 transport protocol
```

amitfernandes committed:
minor updates

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -6,22 +6,27 @@ endif::[]
+
+* Configuration of protcols such as AS2, SFTP, HTTPS
+
+* Understanding of the concept of organizations and environments in the Anypoint platform in context to Access Management and Mule Runtime Manager
```
```suggestion
* Organizations and environments in Anypoint Platform in relation to access management and Anypoint Runtime Manager
```

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -6,22 +6,27 @@ endif::[]
 Verify that you have the following software and other requirements:
 
-* Mule runtime engine 4.1.1 or higher. Make sure you have the needed Mule runtime Core/vCore capacity to handle your B2B transaction workload needs.
+* Mule runtime engine 4.1.1 or higher. Make sure you have the needed Mule runtime Core/vCore capacity to handle your B2B transaction workload needs
```
```suggestion
* Mule runtime engine 4.1.1 or later. 
  Ensure that you have the Mule runtime Core/vCore capacity required to process your B2B transaction workload. If you are uncertain, contact your MuleSoft representative.
```

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -6,22 +6,27 @@ endif::[]
-* Mule runtime engine 4.1.1 or higher. Make sure you have the needed Mule runtime Core/vCore capacity to handle your B2B transaction workload needs.
+* Mule runtime engine 4.1.1 or higher. Make sure you have the needed Mule runtime Core/vCore capacity to handle your B2B transaction workload needs
+
 If you are unsure about the capacity you need to handle your B2B transactions, please contact your MuleSoft account representative for assistance.
```
omit

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -6,22 +6,27 @@ endif::[]
+
 If you are unsure about the capacity you need to handle your B2B transactions, please contact your MuleSoft account representative for assistance.
 
 * A storage solution to store the B2B transmission payloads your send or receive. Storage options can include a Database, Amazon S3, Azure Blob Storage, and so on.
```
```suggestion
* A storage solution capable of storing your anticipated B2B transmission payloads. 
Storage options can include a database, Amazon S3, Azure Blob Storage, and so on.
```

amitfernandes committed:
first pass on the edits

amitfernandes committed:
update # 2

amitfernandes committed:
completed updating the document

amitfernandes committed:
updating indentation

amitfernandes committed:
updated the page

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -3,25 +3,30 @@ ifndef::env-site,env-github[]
 * A storage solution to store the B2B transmission payloads your send or receive. Storage options can include a Database, Amazon S3, Azure Blob Storage, and so on.
 
-* Anypoint Studio 7.1 or higher to create DataWeave maps.
+* Anypoint Studio 7.1 or higher to create DataWeave maps
```
```suggestion
* Anypoint Studio 7.1 or later to create DataWeave maps
```

Already edited this above. I am confused by repeated text. Please do not include it in PRs.

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -39,7 +44,8 @@ NOTE: To obtain an Anypoint Partner Manager entitlement, please contact your Mul
 == Environment Configuration
 
-You'll need to set up and configure your environment before you begin using Partner Manager. Please refer to Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[configuring your Anypoint platform environments].
+You'll need to set up and configure your environment before you begin using Partner Manager.
```
Sounds like it should be just a list item in prerequisites, with a link in See Also.

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -39,7 +44,8 @@ NOTE: To obtain an Anypoint Partner Manager entitlement, please contact your Mul
 
-You'll need to set up and configure your environment before you begin using Partner Manager. Please refer to Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[configuring your Anypoint platform environments].
+You'll need to set up and configure your environment before you begin using Partner Manager.
+Please refer to Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[configuring your Anypoint platform environments].
```
ditto

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -51,17 +57,12 @@ Both roles have unrestricted access, but one role works in a Production environm
 
 . Verify that you have access to Partner Manager.
-This means that you must have the required entitlements and have assigned the `Partner Manager Administrator` role in Access Management for the environment.
+This means that you must have the required entitlements and have assigned the `Partner Manager Admin` role in Access Management for the environment.
```
What is an "entitlement"? Could this simply be You must have the Partner Manager Admin role for access management?

GLOBAL: Do not capitalize access management.

Official name | access management | As a capability, this is a descriptor of the functionality. These are predetermined names for capabilities, so while they are not a name, they should be described consistently
-- | -- | --

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -51,17 +57,12 @@ Both roles have unrestricted access, but one role works in a Production environm
 
-. Setup the required Storage API (to be added: a link to Exchange with description and RAML).
-The storage API should be accessible from the Mule Runtime network.
+. Setup the required Storage API. The storage API should be accessible from the Mule Runtime network.
```
```suggestion
. Set up the required storage API. 
  The storage API should be accessible from the Mule network.
```

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -99,7 +100,7 @@ wrapper.java.maxmemory=3024
 
 === Setup the Payload Storage API
```
GLOBAL: Search for "setup" and evaluate for correct presentation.

setup = noun, adjective
set up = verb

kbostrom commented on modules/ROOT/pages/setup.adoc:
```diff
@@ -99,7 +100,7 @@ wrapper.java.maxmemory=3024
 
-Anypoint Partner Manager enables you to connect your own storage solution (such as a database, Amazon S3, Azure Blob Storage, and so on) to store the transmission content such as incoming and outgoing B2B messages. You'll need to establish an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage. To setup the Storage API, follow the steps indicated b ...
+Anypoint Partner Manager enables you to connect your own storage solution (such as a database, Amazon S3, Azure Blob Storage, and so on) to store the transmission content such as incoming and outgoing B2B message payloads. You will need to establish an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage. To setup the Storage API, follow the steps i ...
 
```
This looks like text that already exists above.
NOT EDITED

kbostrom commented on modules/ROOT/pages/configure-message-flows.adoc:
```diff
@@ -3,52 +3,73 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
```
Has this file been added while I was editing?
If so, that's not goodI need to be commenting on text that is not changing while I am in it.
NOT EDITED

kbostrom reviewed (approved):
Hey Maria, a couple of things:
In the future, I will not be able to edit things that are still under technical review. It is not a good use of my time, especially if other reviewers and yourself are working in the file at the same time that I am. I was very confused by what seemed like duplicate text, or text changed or added during the two days I have been in this file.
Please send me in process text outside of the repo; please send me only "finished" text (that has been through tech review) in a PR.

Okay, sent you a message about this.

amitfernandes committed:
minor language update

amitfernandes committed:
minor formatting updates

amitfernandes committed:
formatting update

amitfernandes committed:
Indentation updates

amitfernandes committed:
content updates # 1

amitfernandes committed:
content update # 2

amitfernandes committed:
content update # 2

amitfernandes committed:
updated indentation

mcherem committed:
Update B2B-overview.adoc

Kim's edits.

mcherem committed:
Update endpoint-http-send.adoc

Remove 'Type' as it does not add value for the user.

mcherem committed:
Update endpoint-http-send.adoc

consistency edits.

mcherem committed:
Update activity-tracking.adoc

Editorial changes to doc style.

mcherem committed:
Update activity-tracking.adoc

Fixed steps.

mcherem committed:
Update configure-host.adoc

Edit to doc style.

mcherem committed:
Update configure-partner.adoc

Fixed to our editorial style.

mcherem committed:
Update configure-partner.adoc

Kim's edits.

mcherem committed:
Update configure-message-flows.adoc

Editorial changes to style and grammar.

mcherem committed:
Update configure-message-flows.adoc

Fixed numbering.

mcherem committed:
Update configure-partner.adoc

Added xref to X12 Global settings.

mcherem committed:
Update x12-receive-read-settings.adoc

Moved info from procedure section back into this section for reference.

mcherem committed:
Update configure-partner.adoc

Moved reference material back into the reference section per Editor.

mcherem committed:
Update configure-host.adoc

Removed reference material (X12 Global Settings) from procedure file per Editor.

mcherem committed:
Update B2B-overview.adoc

Minor edit.

mcherem committed:
Update setup.adoc

Commented out TS until I can check with PM for more info.

mcherem committed:
Fixing index.adoc problems.
